docs/developer/api.rst

```diff
@@ -0,0 +1,349 @@
+The GeoMesa API
+===============
+
+GeoTools DataStore
+------------------
+
+GeoMesa provides multiple classes that implement the `GeoTools <http://geotools.org>`_ |geotools_version| `DataStore <http://docs.geotools.org/latest/userguide/library/api/datastore.html>`_ class, each of which uses a different backing store. These include ``DataStore``\ s that use Accumulo or Kafka.
+
+Accumulo DataStore
+^^^^^^^^^^^^^^^^^^
```

Comment:
I've updated the text in this section of the readme:
https://github.com/locationtech/geomesa/tree/master/geomesa-accumulo/geomesa-accumulo-datastore